### DISCUSSION TITLE
PUBLISHED DATA

### DISCUSSION THREAD

**Discussion by hfatima1** [lisphilar/covid19-sir#1004] [General]
Are there any published papers using this directory? I have analysed my data and need references and an idea about interpretation of data.

**Comment by lisphilar** [comment]
We have no published papers for CovsirPhy at this time. As written in README, please cite this library as follows with version number (`import covsirphy as cs; cs.__version__`).

CovsirPhy Development Team (2020-2022), CovsirPhy version [version number]: Python library for COVID-19 analysis with phase-dependent SIR-derived ODE models, [https://github.com/lisphilar/covid19-sir](https://github.com/lisphilar/covid19-sir)

If you want to use SIR-F model, S-R trend analysis, phase-dependent approach to SIR-derived models, and other scientific method performed with CovsirPhy, please cite the next Kaggle notebook.

Hirokazu Takaya (2020-2022), Kaggle Notebook, COVID-19 data with SIR model, [https://www.kaggle.com/lisphilar/covid-19-data-with-sir-model](https://www.kaggle.com/lisphilar/covid-19-data-with-sir-model)

We can check the citation with the following script.

```Python
import covsirphy as cs
cs.__citation__
```
